### PR TITLE
Check homing feature and dynamic limiting on arm and extension

### PR DESCRIPTION
**Pull request by GearBoxFox**

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -93,17 +93,19 @@ public void robotPeriodic() {
...
     // Checks every 100 milliseconds (roughly 10 robot cycles) to see if any Spark Maxes have rebooted
     // if one has it will then rerun CAN ID configurations on it to stop CAN bus from overflowing
-
     if (m_timer.get() >= 5.0) {
       SparkMaxFactory.Companion.updateCanFramePeriods();
       m_timer.reset();
     }
+
+    // For testing purposes
+    m_robotContainer.getArmSupersystem().calculateArmAngleLimit(20);
```
Comment (Zoarial94) [inline src/main/java/frc/robot/Robot.java:102]: This should not be run while the robot is enabled.

File: src/main/java/frc/robot/commands/HoldArmAngleCommand.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    }
+
+    @Override
+    public void initialize() {
+        m_staticAngle = m_angle.getArmAngle();
+    }
+    @Override
+    public void execute() {
+        m_angle.setArmAngle(m_staticAngle);
+    }
```
Comment (Zoarial94) [inline src/main/java/frc/robot/commands/HoldArmAngleCommand.java:27]: This command should really stop the arm, and then record that angle.

**Review by Zoarial94 — CHANGES_REQUESTED** [review]
I will work on these 2 things

**Comment by GearBoxFox** [comment]
This is all in pull request #98  now

**Pull request closed without merging**